afisha-app/src/app/app.routes.ts

```diff
@@ -8,5 +8,6 @@ export const appRoutes: Routes = [
   { path: 'events', component: EventsListComponent },
   { path: 'events/:id', component: EventComponent, canActivate: [EventRouteActivatorService] },
   { path: '404', component: Error404Component },
-  { path: '', redirectTo:'/events', pathMatch: 'full'}
+  { path: 'user', loadChildren: 'app/user/user.module#UserModule' },

```

Comment:
Move all path strings to PATH const. 
Create /shared folder -> in this folder create /constant folder -> and create path.constant.ts file.
In this file create const PATH with all route paths and use this const instead string.